dev-java/protobuf-java/protobuf-java-3.20.1-r1.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 2008-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Tests not enabled, depend on com.google.truth which is not packaged
+# https://github.com/protocolbuffers/protobuf/blob/v3.20.1/java/core/pom.xml#L35-L40
+JAVA_PKG_IUSE="doc source"
+MAVEN_ID="com.google.protobuf:${PN}:${PV}"
+
+inherit java-pkg-2 java-pkg-simple
+
+PARENT_PN="${PN/-java/}"
+PARENT_PV="${PV}"
+PARENT_P="${PARENT_PN}-${PARENT_PV}"
+
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+
+	EGIT_REPO_URI="https://github.com/protocolbuffers/${PARENT_PN}"
+	EGIT_SUBMODULES=()
+	EGIT_CHECKOUT_DIR="${WORKDIR}/${PARENT_P}"
+else
+	inherit verify-sig
```

Comment:
Why verify-sig? We don't download any signatures nor do we have a key set.

Reply:
I agree I took the liberty to copy this feature from other ebuilds thinking I will be able to enable it.

You are right, I should remove it.

Reply:
Removed in commits under 17693239493746d79429c98b3b83b71d7ed31c50.